src/CMakeLists.txt

```diff
@@ -40,16 +49,14 @@ target_link_libraries(visor-core
         rng
         timer
         opentelemetry_proto
-        ${CONAN_LIBS_PROTOBUF}
-        ${CONAN_LIBS_ABSEIL}
```

Comment:
Just to double check: abseil not needed anymore?

Reply:
it is being properly linked by protobuf lib now